packages/react/src/modules/slack/components/SlackChannelCombobox/ConnectionErrorInfoBoxes.tsx

```diff
@@ -0,0 +1,38 @@
+import { useKnockSlackClient } from "@knocklabs/react-core";
+
+import InfoIcon from "./icons/InfoIcon";
+
+const ConnectionErrorInfoBoxes = () => {
+  const { connectionStatus } = useKnockSlackClient();
+  return (
+    <>
+      {connectionStatus === "disconnected" && (
```

Comment:
Looks like the only difference is the message, so I think these could be combined to:
```
        <div className="rnf-disconnected-info-container">
          <span>
            <InfoIcon />
          </span>

          <div className="rnf-info-container-text">
           {<connection message>}
          </div>
        </div>
 ```
We could either have a conditional in the jsx or a separate function that returns the message.